### ISSUE TITLE
Title of figures

### ISSUE THREAD

**Issue by youngahn** [CCS-Lab/easyml#62]
The title of a figure on model predictions needs to indicate whether predictions are on a training set or a test set.

**Comment by paulhendricks** [comment]
Finished in R as of 017c2a94aa11a21b0e1cd6598cde473f558c659a.

**Comment by paulhendricks** [comment]
Finished in Python as of 40421dae45ab006603b1d82c32bb5fc916fb630a.
